doc/api/fs.md

```diff
@@ -4949,6 +4949,11 @@ The following constants are meant for use with `fs.open()`.
     <td><code>O_NONBLOCK</code></td>
     <td>Flag indicating to open the file in nonblocking mode when possible.</td>
   </tr>
+  <tr>
+    <td><code>UV_FS_O_FILEMAP</code></td>
+    <td>When set, a memory file mapping is used to access the file. This flag
+    is available on Windows operating systems only.</td>
```

Comment:
would be useful to state how it is handled on other systems. Is it ignored? Does it throw? etc.